### PR TITLE
Repurpose context menu

### PR DESCRIPTION
**Pull request by darkwing**
Provides the same context menu for individual column breakpoint markers as the left line/gutter marker.

### CODE REVIEW

**Comment by darkwing** [comment]
Demos:

conditional

context

File: src/components/Editor/GutterMenu.js [hunk 1]
```diff
@@ -139,17 +140,24 @@ class GutterContextMenuComponent extends Component {
...
     const { contextMenu, ...props } = nextProps;
     const { event } = contextMenu;
+
     const sourceId = props.selectedSource ? props.selectedSource.id : "";
     const line = lineAtHeight(props.editor, sourceId, event);
+    const column =
+      props.editor.codeMirror.coordsChar({
+        left: event.x,
+        top: event.y
+      }).ch || undefined;
```
Comment (jasonLaster) [inline src/components/Editor/GutterMenu.js]: do you need ` || undefined;`

Comment (jasonLaster) [reply src/components/Editor/GutterMenu.js]: why?

Comment (darkwing) [reply src/components/Editor/GutterMenu.js]: You're right :(

**Pull request merged**